db/migrate/20240426143215_remove_orphaned_assets.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOrphanedAssets < ActiveRecord::Migration[6.0]
+  def up
+    ForemanOpenscap::AssetPolicy.left_outer_joins(:asset).where(asset: { id: nil })
```

Comment:
Ermmm, sorry about that.

I had to go a slightly roundabout way about it, just calling delete_all on this made activerecord build a nonsensical query
```
>  ForemanOpenscap::AssetPolicy.left_outer_joins(:asset).where(asset: { id: nil }).delete_all
/usr/share/gems/gems/activerecord-6.1.7.7/lib/active_record/connection_adapters/postgresql_adapter.rb:672:in `exec_params': PG::SyntaxError: ERROR:  zero-length delimited identifier at or near """" (ActiveRecord::StatementInvalid)
LINE 1: ...policies" WHERE "foreman_openscap_asset_policies"."" IN (SEL...
```

Full error
```
2024-04-26 10:05:42 EDT STATEMENT:  DELETE FROM "foreman_openscap_asset_policies" WHERE "foreman_openscap_asset_policies"."" IN (SELECT "foreman_openscap_asset_policies"."" FROM "foreman_openscap_asset_policies" LEFT OUTER JOIN "foreman_openscap_assets" "asset" ON "asset"."id" = "foreman_openscap_asset_policies"."asset_id" WHERE "asset"."id" IS NULL)
2024-04-26 10:09:02 EDT ERROR:  zero-length delimited identifier at or near """" at character 87
```

Looks like active record gets confused because about what primary key there should be.